### PR TITLE
[PUBDEV-5411] Remove additional progress bar from AutoML in Python

### PR DESCRIPTION
**Pull request by terrytangyuan**
Ticket: https://0xdata.atlassian.net/browse/PUBDEV-5411
This is similar to what's been done in R https://github.com/h2oai/h2o-3/commit/f7dd5e0294b9928588dff64325df5f3124a5b5b5.

### CODE REVIEW

File: h2o-py/h2o/automl/autoh2o.py [hunk 1]
```diff
@@ -344,8 +344,19 @@ def _fetch(self):
         else:
             self._leader_id = None
 
-        # Parse leaderboard H2OTwoDimTable & return as an H2OFrame
-        leaderboard = h2o.H2OFrame(res["leaderboard_table"].cell_values, column_names=res["leaderboard_table"].col_header)
+        # Intentionally mask the progress bar here since showing multiple progress bars is confusing to users.
+        # If any failure happens, revert back to user's original setting for progress and display the error message.
+        try:
```
Comment (navdeep-G) [inline h2o-py/h2o/automl/autoh2o.py]: Where is the progress bar turned off?

Comment (terrytangyuan) [reply h2o-py/h2o/automl/autoh2o.py]: Oops forgot to push the other commit. Thanks!

**Comment by navdeep-G** [comment]
LGTM

**Comment by terrytangyuan** [comment]
@michalkurka Wanna take a final look before I merge it?

**Comment by michalkurka** [comment]
@terrytangyuan looks good, thank you Terry

**Pull request merged**